docs/UserGuide.md

```diff
@@ -200,14 +200,12 @@ existing Member in the application with the name you specify.
 Format: `editMember NAME [n/NEW_NAME] [p/NEW_PHONE_NUMBER] [e/NEW_EMAIL] [r/NEW_ROLE]`
 
 * Edits the member at the specified NAME.
-* The fields NEW_NAME, NEW_PHONE_NUMBER, NEW_EMAIL, NEW_ROLE are all optional.
 * At least one of the optional fields must be provided.
-* NEW_ROLE field can take on any values with alphanumeric characters.
-* NEW_NAME field can take on any values with alphanumeric characters.    
-* Existing values will be updated to the input values.
+* NEW_NAME and NEW_ROLE fields can take on any values with alphanumeric characters.
+* Existing values will be updated to the input values. 
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-The NAME you specify must be in the currently displayed list of Members 
+The NAME specified is case-sensitive and must be in the currently displayed list of Members 
```

Comment:
Should keep the formatting consistent. Put case sensitive above too or keep it as such